src/app/app.component.html

```diff
@@ -1 +1,4 @@
-<app-flashcard-page></app-flashcard-page>
+<!--<h1>hello</h1>-->
+<!--<app-flashcard-page></app-flashcard-page>-->
```

Comment:
Since the routing module now has logic for displaying the page, remove these commented lines